examples/wind_example.ipynb

```diff
@@ -0,0 +1,925 @@
+      "text/plain": [
+       "<Figure size 432x288 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "fig = plt.figure()\n",
+    "ax = fig.add_subplot(111)\n",
+    "ax.set_xlabel('Time (h)')\n",
+    "ax.set_ylabel('Speed (m/s)')\n",
+    "ax.set_title('Hourly mean wind speeds on January 11, 2018')\n",
+    "line = ax.plot(hours,hourly_mean_speeds,'o')"
```

Comment:
what do you think about adding gridlines to the plot?

Reply:
Good call, it makes the plot easier to read